src/yoroi-wallets/cardano/api/api.ts

```diff
@@ -171,3 +200,31 @@ const isTokenRegistryEntry = (data: unknown): data is TokenRegistryEntry => {
     candidate.name.value === 'string'
   )
 }
+
+export const parseModerationStatus = (status: unknown): YoroiNftModerationStatus | undefined => {
+  const statusString = String(status)
+  const map = {
+    RED: 'blocked',
+    YELLOW: 'consent',
+    GREEN: 'approved',
+    PENDING: 'pending',
+    MANUAL_REVIEW: 'manual_review',
+  } as const
+  return map[statusString.toUpperCase() as keyof typeof map]
+}
+
+function parseNFTs(value: unknown, storageUrl: string): YoroiNft[] {
+  if (!value || typeof value !== 'object') {
+    return []
+  }
+
+  const assets = Object.values(value).map((arrayWithOneAsset) => arrayWithOneAsset[0])
```

Comment:
this introduces an `any`
if you want an array of length 1, throw an error to guarantee it
can you access the value with a key instead?